test/dev/async-config.size-limit.test.js

```diff
@@ -23,7 +23,7 @@ describe('nuxt basic resources size limit', () => {
     const LEGACY_JS_RESOURCES_KB_SIZE = 217
     expect(legacyResourcesSize.uncompressed).toBeWithinSize(LEGACY_JS_RESOURCES_KB_SIZE)
 
-    const LEGACY_JS_RESOURCES_GZIP_KB_SIZE = 70
+    const LEGACY_JS_RESOURCES_GZIP_KB_SIZE = 75
```

Comment:
We've removed url-polyfill, any specific reason that the size is increased ?

Reply:
It is mostly because of standalone encoding utils / ponycode added to ufo bundle (in 0.5.0 i've removed dependency on it when tree-shaking but still normalizeURL would need it)

Reply:
FYI this is entire implementation: https://unpkg.com/@nuxt/ufo@0.5.0/dist/index.js

Reply:
The ufo bundle doesn't look bigger than url-polyfill + 5KB at my first sight.

Reply:
~2.5kB untreeshaked [bundlephobia](https://bundlephobia.com/result?p=@nuxt/ufo@0.5.0) . Legacy bundle:

![image](https://user-images.githubusercontent.com/5158436/102350166-c52a4400-3fa4-11eb-8ed1-808e7c3d6024.png)


Modern:

![image](https://user-images.githubusercontent.com/5158436/102350222-e0954f00-3fa4-11eb-8803-b130d868103a.png)


Conclusion: Overhead is mostly babel transpilation. I will try to see if we can use some hacks to avoid or reduce it

Reply:
BTW gzip diff is +1KB. It is strange with tests too....

Reply:
You can remove the range and see real diff https://github.com/nuxt/nuxt.js/blob/7d18b06f15bcbd9f466335cc695f71177a6add6e/test/dev/async-config.size-limit.test.js#L9

Reply:
@clarkdo Regardless of tests, i think _overhead_ is correct from bundle analyzer :D 2-3KB/gzip overhead of ufo is almost unavoidable and by using polyfills inside ufo to reduce transpilation overhead, we might add overhead to modern build.

Reply:
>  2-3KB/gzip overhead of ufo

This looks good to me. I agree that not adding polyfills in ufo package.